beancount_reds_importers/libtransactionbuilder/investments.py

```diff
@@ -96,8 +96,7 @@ def get_target_acct(self, transaction):
     def get_security_list(self):
         tickers = set()
         for ot in self.get_transactions():
-            if ot.type in ['buymf', 'sellmf', 'buystock', 'sellstock', 'reinvest', 'income']:
-                tickers.add(ot.security)
+            tickers.add(ot.security)
```

Comment:
The trouble with relying upon KeyError is, you can get those only one by one, which is a very painful way to do it when adding a new account with say, 25 securities. This is unfortunately a case which is hard for me to debug because I don't seem to have those types. If you could find out why your modification to the list above doesn't work as expected, that would be awesome.

Reply:
I'm going to leave this PR open, and treat it as a bug until resolved. Let me try to run with your data above to see if I can reproduce it.